### ISSUE TITLE
Replace `future` with `six`

### ISSUE THREAD

**Issue by guillochon** [astrocatalogs/astrocats#80]
This will remove a package dependency on `future`, which few people will have installed.

**Comment by lzkelley** [comment]
`future` (as in https://pypi.python.org/pypi/future) is used in places, instead of `__future__` (https://docs.python.org/2/library/__future__.html)?

**Comment by guillochon** [comment]
Correct, it's actually imported as `past`, which is a part of the `future` package. It's mostly for string-related issues.
